src/bokeh/sampledata/emissions.py

```diff
@@ -0,0 +1,96 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2023, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+''' CO2 emmisions of selected countries in the years 2000 and 2010.
+
+License: `Public Domain`_
+
+This module contains one pandas Dataframe: ``data``.
+
+.. rubric:: ``data``
+
+:bokeh-dataframe:`bokeh.sampledata.emissions.data`
+
+.. bokeh-sampledata-xref:: emissions
+'''
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+from typing import TYPE_CHECKING
+
+# Bokeh imports
+from ..util.sampledata import package_csv
+
+if TYPE_CHECKING:
+    import pandas as pd
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    'data',
+)
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Dev API
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Private API
+#-----------------------------------------------------------------------------
+
+def _read_data() -> pd.DataFrame:
+    df = package_csv('emissions','emissions.csv')
```

Comment:
@Azaya89 I would stop here, more or less :) Doing all the processing below just precludes other users from using the full dataset in case they would like to do that.  If there are simple type-conversions or other cleanup, that could be ok, but I would return a full dataframe for all raw data and leave it to examples or users to do any filtering or joining, etc.

Reply:
OK